docs/source/upgrade-guide/index.md

```diff
@@ -474,6 +474,35 @@ The recommended way of generating a project boilerplate is [Cookieplone](https:/
 Please update your code to use the `pnpm` based setup.
 ```
 
+### Update needed to project boilerplate generated with `@plone/generator-volto`
+
+```{versionadded} Volto 18.0.0-alpha.42
+From this version, a new feature introduced a breaking change in the boilerplates created using `@plone/generator-volto` previous to version 9.0.0-alpha.17.
```

Comment:
This sentence does not make sense to me because of the various versions implied but not explicitly stated. Is this what you mean?

```suggestion
Effective with Volto 18.0.0-alpha.42, a new feature introduced a breaking change in the boilerplates created using `@plone/generator-volto` 9.0.0-alpha.17 and earlier.
```